todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java

```diff
@@ -73,16 +75,15 @@ public void registerIdlingResource() {
 
     @Test
     public void errorShownOnEmptyTask() {
-        // Add task title and description
+        // Add invalid title and description combination
         onView(withId(R.id.add_task_title)).perform(typeText(""));
         onView(withId(R.id.add_task_description)).perform(typeText(""),
                 closeSoftKeyboard());
-        // Save the task
+        // Try to save the task
         onView(withId(R.id.fab_edit_task_done)).perform(click());
 
-        // Verify empty tasks snackbar is shown
-        String emptyTaskMessageText = getTargetContext().getString(R.string.empty_task_message);
-        onView(withText(emptyTaskMessageText)).check(matches(isDisplayed()));
+        // Verify that the activity is still displayed (a correct task would close it).
+        onView(withId(R.id.add_task_title)).check(matches(isDisplayed()));
```

Comment:
Here you're verifying that the activity is still displayed. The test is called "errorShownOnEmptyTask" but there's no verification that an error is shown.